lib/Cmts.mli

```diff
@@ -55,6 +55,19 @@ val relocate :
     locations) comments before [src] to [before] and comments after [src] to
     [after]. *)
 
+val relocate_match_cmts :
```

Comment:
This is not only  used for `match`, can we find a better name ? (and update the doc comment)

Reply:
I've renamed the function `relocate_pattern_matching_cmts` and improved the doc comment